getting-started/multiband-analysis.rst

```diff
@@ -101,7 +101,8 @@ Filtering for unique, deblended sources with the detect_isPrimary flag
 
 Before going ahead and plotting a CMD from the full source table, you'll typically need to do some basic filtering.
 Exactly what filtering is done depends on the application, but source tables should *always* be filtered for unique sources.
-There are two ways that measured sources might not be unique: deblended sources, and sources in patch overlaps.
+There are two ways that measured sources might not be unique: deblended sources, and sources in patch
```

Comment:
Suggested re-write:

```
There are two ways that measured sources might not be unique: deblended sources, and sources in patch overlaps.
Additionally, some sources are "sky" objects added by ``detectCoaddSources.py`` for noise characterization that you need want to filter out.
```

(Two sentences instead of a semicolon is easier to read).
Be careful not to hardwrap your text content and use one-sentence per line instead.